worker/firewaller/firewaller_test.go

```diff
@@ -58,7 +59,8 @@ type firewallerBaseSuite struct {
 	crossmodelFirewaller *crossmodelrelations.Client
 	clock                clock.Clock
 
-	callCtx context.ProviderCallContext
+	callCtx     context.ProviderCallContext
+	credentials *credentialvalidator.Facade
```

Comment:
Perhaps call this credentialFacade?
To avoid ambiguity when people read the code using this and think it refers to an actual credential